.github/workflows/release-python.yml

```diff
@@ -30,15 +31,22 @@ jobs:
         - [ubuntu-20.04, manylinux_x86_64]
         - [macos-11, macosx_*]
         - [windows-2019, win_amd64]
-        python: ["cp37", "cp38", "cp39", "cp310"]
+        python: ["cp37", "cp38", "cp39", "cp310", "cp311"]
 
     steps:
       - name: Checkout pymongoarrow
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-python@v3
+      - name: Set up python version
+        run: |
+          export PYTHON_VERSION=$(sed 's/^cp3/3./' <<< ${{ matrix.python }} )
+          echo "PYTHON_VERSION=$PYTHON_VERSION" >> $GITHUB_ENV
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{env.PYTHON_VERSION}}
```

Comment:
Is this needed? I thought this python was only used to run cibuildwheel itself and then cibuildwheel took care of the rest of the work to generate wheels for each python version on each platform.

Reply:
We needed it because we aren't using the cibuildwheel action, since we needed to tweak a few things.